hack/build-client.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+BUILD_WORKDIR="${PWD}"
+
+function clean() {
+  rm ${BUILD_WORKDIR}/.sha256sum
+
+  rm -rf ${BUILD_WORKDIR}/pkg/client
+  mkdir ${BUILD_WORKDIR}/pkg/client
+
+  rm -rf ${BUILD_WORKDIR}/python/client
+  mkdir -p ${BUILD_WORKDIR}/python/client
+}
+
+function checksum() {
+  sha256sum swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/"$1"/redhatopenshift.json >> .sha256sum
+}
+
+function generate_golang() {
+  sudo docker run \
+		--rm \
+		-v ${BUILD_WORKDIR}/pkg/client:/github.com/Azure/ARO-RP/pkg/client:z \
+		-v ${BUILD_WORKDIR}/swagger:/swagger:z \
+		azuresdk/autorest \
```

Comment:
I'm wondering if `azuresdk/autorest` can actually generate clients for multiple versions in one run. Looking at [docs](https://github.com/Azure/autorest/blob/master/docs/user/command-line-interface.md#commonly-used-settings) and this:

> Adds the given file to the list of input files for generation process

sounds like you can privide `--input-file`s.

Reply:
So I have played with the autorest and it says input list, but it does not support multiple inputs.

e.g.
```
FATAL: swagger-document-override/md-override-loader - FAILED
FATAL: Error: Could not read 'file:///swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2020-04-30/redhatopenshift.json,/swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2020-12-31-preview/redhatopenshift.json,'.
FATAL: swagger-document/loader - FAILED
FATAL: Error: Could not read 'file:///swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2020-04-30/redhatopenshift.json,/swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2020-12-31-preview/redhatopenshift.json,'.
Process() cancelled due to exception : Could not read 'file:///swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2020-04-30/redhatopenshift.json,/swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2020-12-31-preview/redhatopenshift.json,'.
  Error: Could not read 'file:///swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2020-04-30/redhatopenshift.json,/swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2020-12-31-preview/redhatopenshift.json,'.
```

I have tried splitting with space, with a colon. Moreover, for the golang package, it does not support generating multiple outputs, and our code is organized in multiple folders.

Therefore I would leave it as it is.